alacritty/src/window.rs

```diff
@@ -254,22 +262,26 @@ impl Window {
 
         let class = &window_config.class;
 
-        let mut builder = WindowBuilder::new()
+        let builder = WindowBuilder::new()
```

Comment:
Why did you remove the mutability and then later just redefine the variable? Wouldn't it be easier to just mutate it?

Reply:
Rust doesn't allow putting cfg before assignment.

Reply:
Huh, you're right. TIL.